src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
```

Comment:
you should pass color as a parameter too and initialize it using the constructor of Figure, the same with other figures